docs/reference/indices/open-close.asciidoc

```diff
@@ -32,3 +32,7 @@ This setting can also be changed via the cluster update settings api.
 
 Closed indices consume a significant amount of disk-space which can cause problems in managed environments. Closing indices can be disabled via the cluster settings
 API by setting `cluster.indices.close.enable` to `false`. The default is `true`.
+
+Closed index responses with `index_closed_exception` and `403` for most APIs, including `_stats` and `_search`. `ignore_unavailable` allows to control whether closed index do not cause `index_closed_exception` to be returned.  Index returns value on call to `_aliases`. 
+
+Closed indices are not returned in most wildcard index API calls, like `_aliases`, `_stats` or `_search`, nor throw any exceptions and behave like non-existent.
```

Comment:
I am not sure about this sentence. You control through ignore_unavailable whether closed indices throw error or not. They are unavailable indices for most of the APIs, hence either they are ignored or they cause an error (e.g. it is not technically possible to search against a closed index, so the question is just what to do instead: error or skip them?)